src/test/java/com/wikia/webdriver/common/users/FacebookTestUser.java

```diff
@@ -0,0 +1,14 @@
+package com.wikia.webdriver.common.users;
+
+import com.wikia.webdriver.common.core.XMLReader;
+import com.wikia.webdriver.common.core.configuration.Configuration;
+
+public class FacebookTestUser {
+  private static final String envType = Configuration.getEnvType();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00115)